ui/admin/app/components/targets/target/actions/index.hbs

```diff
@@ -15,6 +15,16 @@
       {{t 'resources.target.actions.add-brokered-credential-sources'}}
     </dropdown.link>
   {{/if}}
+
+  {{#if (can 'addInjectedApplicationCredentialSources target' @model)}}
```

Comment:
Has the convention always been to use the method name and not the string form (i.e. `can 'add injected application credential sources for target' @model`) that the [ember can](https://github.com/minutebase/ember-can#looking-up-abilities) convention describes?

Reply:
not sure why we chose this. Randy would know more about this. I personally like the camel case convention that we currently follow.